wandb/sdk/internal/sender.py

```diff
@@ -880,6 +880,27 @@ def _sync_spell(self) -> None:
             pass
         # TODO: do something if sync spell is not successful?
 
+    def _setup_fork(self, server_run: dict):
+        assert self._settings.fork_from
+        assert self._settings.fork_from.metric == "_step"
+        assert self._run
+        first_step = int(self._settings.fork_from.value) + 1
+        self._resume_state.step = first_step
+        self._resume_state.history = server_run.get("historyLineCount", 0)
+        self._run.forked = True
+        self._run.starting_step = first_step
+
+    def _handle_error(
+        self, record: "Record", error: "wandb_internal_pb2.ErrorInfo", run: "RunRecord"
+    ) -> None:
```

Comment:
nit: add a trailing comma so that the formatter aligns these better

```suggestion
    def _handle_error(
        self,
        record: "Record",
        error: "wandb_internal_pb2.ErrorInfo",
        run: "RunRecord",
    ) -> None:
```